repair/repair.cc

```diff
@@ -2344,7 +2350,18 @@ future<> repair::tablet_repair_task_impl::run() {
                 task_impl_ptr->neighbors = std::move(neighbors);
                 auto task = co_await rs._repair_module->make_task(std::move(task_impl_ptr), parent_data);
                 task->start();
-                co_await task->done();
+                auto res = co_await coroutine::as_future(task->done());
+                if (res.failed()) {
+                    auto ep = res.get_exception();
+                    rlogger.warn("repair[{}]: Repair tablet for table={}.{} range={} status=failed: {}", id.uuid(), m.keyspace_name, m.table_name, m.range, ep);
+                    // keyspace / table still exist?
+                    if (rs.get_db().local().get_tables_metadata().contains(m.tid)) {
```

Comment:
It could be that remote has dropped the table but local node hasn't known it yet.  There is a table_sync_and_check helper does the job.

Reply:
sounds good.  I'll look into it